nservicebus/handlers/accessing-data.md

```diff
@@ -30,6 +48,19 @@ Synchronized storage session by itself only guarantees that there will be no *pa
 
 However, the synchronized storage session **does not guarantee that each state change is persisted exactly once**. To ensure *exactly-once* message processing, synchronized storage session has to support a de-duplication strategy.
 
+### Object/Relational Mappers
+
+When using an Object/Relational Mapper (ORM) like Entity Framework for data access, there is the ability to either inject the data context object via dependency injection or create a data context on the fly and reuse the connection.
+
+#### Create data context
```

Comment:
Seems overkill to have these headers, especially when one of the two options is "don't do that"